diesel/src/query_builder/functions.rs

```diff
@@ -37,6 +35,33 @@ use super::insert_statement::Insert;
 /// # #[cfg(not(feature = "postgres"))]
 /// # fn main() {}
 /// ```
+///
+/// To update multiple columns, give `set` a tuple argument:
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("src/doctest_setup.rs");
+/// #
+/// # table! {
+/// #     users {
+/// #         id -> Integer,
+/// #         name -> VarChar,
+/// #         surname -> VarChar,
```

Comment:
Sorry, but this users table doesn't have a surname column (see `doctest_setup.rs`).